tests/acceptance/lazy-routeable-engine-test.js

```diff
@@ -48,11 +48,12 @@ moduleForAcceptance('Acceptance | lazy routable engine', {
 });
 
 function verifyInitialBlogRoute(assert, loadEvents, application) {
-  assert.equal(loadEvents.length, 4, 'loaded 4 assets');
+  assert.equal(loadEvents.length, 5, 'loaded 5 assets');
   assert.deepEqual(loadEvents[0].split(SEPARATORS), [ '', 'engines-dist', 'ember-blog', 'assets', 'engine-vendor.css' ], 'loaded engine vendor css');
   assert.deepEqual(loadEvents[1].split(SEPARATORS), [ '', 'engines-dist', 'ember-blog', 'assets', 'engine-vendor.js' ], 'loaded engine vendor js');
   assert.deepEqual(loadEvents[2].split(SEPARATORS), [ '', 'engines-dist', 'ember-blog', 'assets', 'engine.css' ], 'loaded engine css');
   assert.deepEqual(loadEvents[3].split(SEPARATORS), [ '', 'engines-dist', 'ember-blog', 'assets', 'engine.js' ], 'loaded engine js');
+  assert.deepEqual(loadEvents[4].split(SEPARATORS), [ '', 'engines-dist', 'ember-blog', 'config', 'environment.js' ], 'loaded environment js');
```

Comment:
Hmm, looks like this is loading the new `config/environment.js` file that we use for Node land. This is sub-optimal as it shouldn't be needed in the browser.

Seems like we have a couple options:
1. Put the environment config file elsewhere in the output so it doesn't get included in the asset manifest
2. Add an option to the asset manifest generator to ignore the environment config (would require changes to [`ember-asset-loader`](https://github.com/ember-engines/ember-asset-loader))
3. Add an additional build step to remove the environment config entry from the generated manifest

@rwjblue do you have any preference here? I'm leaning towards option #2.